.eslintrc

```
@@ -21,10 +21,11 @@
     "quotes": ["error", "double"],
     "no-console": "warn",
     "tailwindcss/no-arbitrary-value": "error",
+    "tailwindcss/classnames-order": "off",
     "tailwindcss/no-custom-classname": [
       "error",
       {
-        "whitelist": ["g\\-yt.+", "legend", "dark"],
+        "whitelist": ["g\\-yt.+", "prose-.+", "legend", "dark"],
       },
```

I'd add text-3xl/9 to the white list explicitly instead of using "classNames"


```suggestion
        "whitelist": ["g\\-yt.+", "prose-.+", "legend", "text-3xl/9","dark"],
      },
```

---

Done